### PR TITLE
Lexer lint fixes and test case for Ignore(). 

### PR DESCRIPTION
**Pull request by superfrink**

### CODE REVIEW

File: lexer_test.go [hunk 1]
```diff
@@ -371,10 +383,96 @@ func TestLexer_Find(t *testing.T) {
...
+	t.Parallel()
+
+	t.Run("basic", func(t *testing.T) {
+		t.Parallel()
+
+		l := NewLexer(runeio.NewReader(strings.NewReader("Hello\n!Ignore!\n")), &wordState{})
+
+		advanced, err := l.Advance(7)
+		if err != nil {
+			t.Errorf("unexpected error: %v", err)
```
Comment (ianlewis) [inline lexer_test.go:396]: nit: This should be `t.Fatalf`

File: lexer_test.go [hunk 2]
```diff
@@ -371,10 +383,96 @@ func TestLexer_Find(t *testing.T) {
...
+		}
+		if got, want := advanced, 7; got != want {
+			t.Errorf("Advance: want: %v, got: %v", want, got)
+		}
+
+		rns, err := l.Peek(7)
+		if err != nil {
+			t.Errorf("unexpected error: %v", err)
```
Comment (ianlewis) [inline lexer_test.go:404]: nit: This should be `t.Fatalf`

File: lexer_test.go [hunk 3]
```diff
@@ -371,10 +383,96 @@ func TestLexer_Find(t *testing.T) {
...
+
+		if got, want := l.Column(), 1; got != want {
+			t.Errorf("Column: want: %v, got: %v", want, got)
+		}
+
+		l.Ignore()
+
+		advanced, err = l.Advance(7)
+		if err != nil {
+			t.Errorf("unexpected error: %v", err)
```
Comment (ianlewis) [inline lexer_test.go:426]: nit: This should be `t.Fatalf`

File: lexer_test.go [hunk 4]
```diff
@@ -371,10 +383,96 @@ func TestLexer_Find(t *testing.T) {
...
+		}
+		if got, want := advanced, 7; got != want {
+			t.Errorf("Advance: want: %v, got: %v", want, got)
+		}
+
+		rns, err = l.Peek(1)
+		if err != nil {
+			t.Errorf("unexpected error: %v", err)
```
Comment (ianlewis) [inline lexer_test.go:434]: nit: This should be `t.Fatalf`

**Review by ianlewis — APPROVED** [review]
LGTM

File: lexer_test.go (hunk 1 above)
Comment (superfrink) [reply lexer_test.go:396]: Should I push a change to this branch or just make the change part of a later PR?

Comment (ianlewis) [reply lexer_test.go:396]: Your call. Usually these kind of changes are easy enough to do that you can do it on that PR but I'll just go ahead and merge this one so it's not held up. Feel free to submit a second PR if you have the bandwidth

Comment (superfrink) [reply lexer_test.go:396]: I'll merge this first and change those in a later PR.

**Pull request merged**